XenAdmin/Wizards/NewSRWizard_Pages/Frontends/LVMoHBA.cs

```diff
@@ -94,7 +99,11 @@ public override void PageLeave(PageLoadedDirection direction, ref bool cancel)
             {
                 LvmOhbaSrDescriptor descr = CreateSrDescriptor(device);
 
-                var action = new SrProbeAction(Connection, master, SrType, descr.DeviceConfig);
+                // create an LVM descriptor, to be used with sr.probe
+                LvmOhbaSrDescriptor lvmdescr = CreateLvmSrDescriptor(device);
+                SR.SRTypes srType = lvmdescr is FcoeSrDescriptor ? SR.SRTypes.lvmofcoe : SR.SRTypes.lvmohba; //srType is a workaround instead of SrType
+
+                var action = new SrProbeAction(Connection, master, srType, descr.DeviceConfig);// TODO: use SRType and descr

```

Comment:
This should be `var action = new SrProbeAction(Connection, master, srType, lvmdescr.DeviceConfig); // TODO: use SRType and descr`

(4th parameter: `lvmdescr `instead of `descr `)